java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/elements/ElementNode.java

```diff
@@ -219,7 +219,7 @@ public void selectionChanged() {
         fireIconChange(); // TODO: Strange
     }
 
-    public Description getDescritption() {
+    public Description getDescription() {
```

Comment:
note to self: similar here, package is not exported. However the difference is that `Description` is public.